### PR TITLE
adding php language

### PR DESCRIPTION
**Pull request by irsayvid**
**Description**

This PR fixes: N/A

**Notes for Reviewers:**

**[Signed commits](https://github.com/DevsTrons/hello-world/blob/master/CONTRIBUTING.md)**

- [x] Yes, I am a Member of **DEVSTRONS'**

### CODE REVIEW

**Review by shuklaritvik06 — APPROVED** [review]
LGTM! Thanks for the contribution!

**Comment by vsnthdev** [comment]
Kindly, finish writing the full article with details like:

1. Why use PHP?
2. Where is PHP used?
3. Popular frameworks in PHP?
4. Links to well-written projects in PHP
5. How to get started and set up a dev environment to write PHP code, etc.

We do not accept code snippets without the articles.

**Comment by irsayvid** [comment]
It wasn't mentioned in the contributing guidelines. No issues. Should I write in my preferred platform and link it here or can I link a blog which already has the required information?

**Comment by vsnthdev** [comment]
As the project is currently in active development, the contributing guidelines aren't fully written yet, my apologies for that.
You should edit the markdown file you have submitted and write a genuine article of your own there. In other words, write the in the markdown file and update this PR.

**Comment by irsayvid** [comment]
@vsnthdev review this and let me know if any further changes are needed

File: lang/php.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+title: PHP is popular!
+author: irsayvid
+---
+
+```php
+<?php
+echo "Hello World!";
+?>
+```
+Docs: PHP is a popular general-purpose scripting language that is especially suited to web development. It is a widely-used, free, and efficient alternative to competitors such as Microsoft's ASP. It is powerful tool for making web pages dynamic and interactive. It is very efficient and can serve millions of hits per day. It is easy to integrate and suports many databases such as MySQL. It is easy to learn and uses both procedural and oject oriented programming. Using php, it is easy to modify database and restrict access to differrent pages for different users. Laravel and Codeigniter are two popular frameworks in php. [Here](https://hackr.io/blog/php-projects) is a blog for users to get started with php, how to setup and write your first hello world in php. It contains few interesting projects you can do in php.
```
Comment (Panquesito7) [inline lang/php.md:13]: A lot of grammar and spelling issues.
```suggestion
Docs: PHP is a popular general-purpose scripting language that is especially suited to web development. It is a widely-used, free, and efficient alternative to competitors such as Microsoft's ASP. It is a powerful tool for making web pages dynamic and interactive. It is very efficient and can serve millions of hits per day. It is easy to integrate and supports many databases such as MySQL. It is easy to learn and uses both procedural and object-oriented programming. Using PHP, it is easy to modify the database and restrict access to different pages for different users. Laravel and Codeigniter are two popular frameworks in PHP. [Here](https://hackr.io/blog/php-projects) is a blog for users to get started with PHP, how to set up and write your first hello world in PHP. It contains few interesting projects you can do in PHP. 

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by Panquesito7** [comment]
The [**contributing guidelines**](https://github.com/devstrons/hello-world#%EF%B8%8F-contribution-guidelines) were updated now. You can update your PR properly now. 🙂

**Comment by irsayvid** [comment]
@vsnthdev @Panquesito7  please review the readme file, proofread the article and suggest if any changes are needed.

File: lang/php.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+A — Apache HTTP Server
+M — MariaDB (formerly MySQL)
+P — PHP
+P — PERL
+
+Additionally, XAMPP comes with other useful tools such as Mercury mail server, phpMyAdmin for database administration, Webalizer for web server log analysis, OpenSSL, Apache Tomcat for running Java code, and FileZilla FTP server.
+Currently, the latest stable version of PHP is 7.4
+
+**Step 1:**
+Navigate to https://www.apachefriends.org/download.html and download the latest stable version of PHP according to your OS (windows/Linux/OSX)
```
Comment (vsnthdev) [inline lang/php.md:50]: Change line 50 to 👇 

```
[Click here](https://www.apachefriends.org/download.html) to download the latest stable version of PHP according to your OS (windows/Linux/OSX)
```

File: lang/php.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
+---
+lang: PHP
+syntax: php
+title: PHP is popular!
+author: irsayvid
+---
+
+```php
+<?php
+echo "Hello World!";
```
Comment (vsnthdev) [inline lang/php.md:10]: Indent this line with 4 spaces.

File: lang/php.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+**Step 3:**
+In the control panel, click on config button. Click on the start button beside Apache and MySQL modules. You can access your Apache server’s dashboard by clicking on the admin button beside it. Alternatively, you can also reach it via http://localhost/dashboard/ URL in your browser.
+You can access your phpMyAdmin dashboard by clicking the admin button beside MySQL module. Or you can simply go to http://localhost/phpmyadmin/ in your browser. Here, you can manage the MariaDB (or MySQL) databases of your web projects.
+
+**Writing your first hello world program in php:**
+Create a new folder hello in C:\xampp\htdocs\ (The C: drive may vary based on the location you chose for installation). Inside the new folder, right click and create a new file ‘hello.php’. Write the below code in your new file.
+
+```php
+<?php
+echo "Hello World!";
```
Comment (vsnthdev) [inline lang/php.md:65]: Indent this line with 4 spaces.

File: lang/php.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+?>
+```
+
+Navigate to  http://localhost/hello/hello.php in your preferred browser.
```
Comment (vsnthdev) [inline lang/php.md:69]: Change line 69 to 👇 

```
Navigate to [http://localhost/hello/hello.php](http://localhost/hello/hello.php) in your preferred browser.
```

File: lang/php.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+> *Congratulations on making it till the end!*
+
+If you’re looking for some projects in PHP for inspiration or contribution, checkout the below projects:
+
+- [Project Celadon](https://github.com/Webwiznitr/Project-celadon-2.0)
+- [Project Sirius](https://github.com/CYBORG-NIT-ROURKELA/project-sirius)
+- [Top open source projects in PHP]( https://github.com/trending/php)
+
+# PHP is case insensitive!
```
Comment (vsnthdev) [inline lang/php.md:81]: Remove this heading and this point anywhere after line 37 (while setting up the dev environment.

**Review by vsnthdev — CHANGES_REQUESTED** [review]
You've done an awesome job @irsayvid ⚡ keep it up!

**Comment by irsayvid** [comment]
I've updated. Please check it once again.

**Comment by vsnthdev** [comment]
**Additional modifications to be done to this document:**

1. Add an SEO friendly title
2. Indent the first code block properly so it looks good

File: lang/php.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+A PHP framework provides a basic structure for streamlining the development of web apps. We use them because they speed up the development process.
+
+**Listed below are few popular PHP frameworks:**
+
+- Laravel
+- CodeIgniter
+- Symfony
+- CakePHP
+
+## Setting up PHP environment locally:
```
Comment (Panquesito7) [inline lang/php.md:35]:
```suggestion
## Setting up PHP environment locally
```

File: lang/php.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
+---
+lang: PHP
+syntax: php
+title: PHP is popular!
+author: irsayvid
+---
+
+```php
+<?php
+echo "Hello World!";
```
Comment (Panquesito7) [inline lang/php.md:10]: I think this missed a 4-line indentation?

**Pull request merged**